packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -30,14 +39,14 @@ import {
   floatingLocalVideoModalStyle,
   SMALL_HORIZONTAL_GALLERY_TILE_SIZE_REM,
   videoGalleryContainerStyle,
-  videoGalleryContainerTokens,
   videoGalleryOuterDivStyle
 } from './styles/VideoGallery.styles';
 import { isNarrowWidth, useContainerWidth } from './utils/responsive';
 import { LocalScreenShare } from './VideoGallery/LocalScreenShare';
 import { RemoteScreenShare } from './VideoGallery/RemoteScreenShare';
 import { VideoTile } from './VideoTile';
 import { v4 as uuidv4 } from 'uuid';
+import { useId } from '@uifabric/react-hooks';
```

Comment:
Is there a reason we use uifabric over fluentui package? https://www.npmjs.com/package/@fluentui/react-hooks#useid

Reply:
Thanks for pointing this out James. Good catch